src/entity_recognizer/EntityDictionary.cpp

```diff
@@ -200,7 +200,7 @@ void EntityDictionary::ReadGazetteerFiles() {
           }
           int l = -1;
           for (l = 0; l < gazetteer_word_entity_tags_[word_id].size();
-               ++l) {
+          ++l) {
```

Comment:
Indeed... but MSVC does seem to have a bug regarding indentation in multi-line 'for' statements, because the 'for' second line is not indented, but the third one is.